markdown/org/docs/faq/tapering-seams/en.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Why do my seam allowances taper on my patterns?
```

Comment:
nit: suggest "Why are seam allowances tapered?"
Also this is great to know! I didn't notice before 😅